Source/common/SantaCacheTest.mm

```diff
@@ -455,4 +455,93 @@ - (void)testContains {
   }));
 }
 
+- (void)testForeach {
+  SantaCache<uint64_t, uint64_t> sut;
+  sut.set(1, 11);
+  sut.set(2, 22);
+  sut.set(3, 33);
+  sut.set(4, 44);
+  sut.set(5, 55);
+
+  sut.set(6, 66);
+  sut.remove(6);
+
+  NSSet *wantKeys = [[NSSet alloc] initWithArray:@[ @(1), @(2), @(3), @(4), @(5) ]];
+  NSSet *wantVals = [[NSSet alloc] initWithArray:@[ @(11), @(22), @(33), @(44), @(55) ]];
+  __block NSMutableSet *gotKeys = [[NSMutableSet alloc] init];
+  __block NSMutableSet *gotVals = [[NSMutableSet alloc] init];
```

Comment:
Why 2 pairs of sets instead of 2 dicts?

Reply:
Heh, cause I didn't really think about it :) Fixed.